### ISSUE TITLE
fetching  remote mermaid.js

### ISSUE THREAD

**Issue by frehberg** [mersinvald/aquamarine#27]
The generated html document contains primary and secondary source of mermaid.js (see below), but as the mermaid.js is not present in folder target/doc/{CRATENAME}, the secondary remote source is used always. 

The doc files should not fetch JS code from remote.   To solve the issue the aquamarine macro-processor should copy  an embedded mermaid.js into the folder target/doc/{CRATENAME}

```
<script src="../mermaid.min.js"></script>

<script>window.mermaid || document.write('<script src="https://unpkg.com/mermaid@9.1.5/dist/mermaid.min.js" crossorigin="anonymous"><\/script>')</script>
```

**Comment by frehberg** [comment]
Fixing:
If invoking "cargo doc --no-deps" the file target/doc/mermaid.min.js is created now
If aquamarine macro is used in sub-module, the HTML rustdoc will be structured as sub-folder. Within the proc-maco we cannot tell length of module path. Therefor in HTML document the script-element must try to load from parent folders until succes, ../, then ../../ etc,

Other fix: glob-pattern /** not supported by cargo:rerun-if-changed in build.rs As specified file './doc/**' does not exist, the module would be rebuilt always!

**Comment by frehberg** [comment]
Ping? Any comments? Is the patch not suitable? Would you like a change?

**Comment by mersinvald** [comment]
Fixed in #29 

Thank you, @frehberg, sorry for the delay on my part
